content/learn/getting-started.md

```diff
@@ -27,19 +26,23 @@ cd calc
 go mod init calc
 ```
 
-Next make sure the Goa module is installed and up-to-date:
+Next install Goa:
 
 ```bash
-go get -u goa.design/goa/v3
-go get -u goa.design/goa/v3/...
+go install goa.design/goa/v3/cmd/goa@v3
```

Comment:
The version of `goa` tool must be the same as `goa.design/goa/v3` module used by the procject, right? I install `goa` like below:

```
go install goa.design/goa/v3/cmd/goa
```

Reply:
Yes if the `goa` package is already in the modules then you don't need to specify the version. 

This is the flow I typically use to create a new Goa service:
```
go mod init <pkg>
mkdir design
vi design/design.go # Create design
goa gen <pkg>/design # Requires `goa` tool
go mod tidy
```

This automatically sets up all the dependencies but for this to work Goa has to be installed globally which requires specifying the version.

Reply:
I see, this is a matter of taste.

I usually do `go get goa.design/goa/v3` before writing design because I want to use suggestions in the editor,  but of course your way is also correct.

Reply:
That makes sense!